samples/SkijaInjectSample/src/main/kotlin/SkijaInjectSample/SkiaPanel.kt

```diff
@@ -1,11 +1,12 @@
-package org.jetbrains.skiko
+package SkijaInjectSample
```

Comment:
If you think the `skiko` library shouldn't have `SkiaPanel.kt`, why didn't you migrate `SkiaWindow.kt` as well?

Reply:
SkiaPanel has issues. It doesn't dispose layer, and probably doesn't support re-adding to the parent component.

SkiaWindow is simple and doesn't have issues.